### PR TITLE
fixes #615

### LINKED ISSUE #615: post install error
I'm trying:

```
> mkdir test
> cd test
> npm init -y
> npm install --save hoodie
```

and getting:

```
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated tough-cookie@2.2.2: ReDoS vulnerability parsing Set-Cookie https://nodesecurity.io/advisories/130

> hoodie@24.3.0 postinstall /Users/bruno/workspace/test_hoodie_cli_2/node_modules/hoodie
> node ./bin/setup.js

fs.js:540
  binding.open(pathModule._makeLong(path),
          ^

TypeError: path must be a string
    at TypeError (native)
    at Object.fs.open (fs.js:540:11)
    at Object.fs.writeFile (fs.js:1132:6)
    at /Users/bruno/workspace/test_hoodie_cli_2/node_modules/hoodie/bin/setup.js:50:10
    at FSReqWrap.oncomplete (fs.js:82:15)
npm WARN test_hoodie_cli_2@1.0.0 No description
npm WARN test_hoodie_cli_2@1.0.0 No repository field.
npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "--save" "hoodie"
npm ERR! node v4.4.7
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE

npm ERR! hoodie@24.3.0 postinstall: `node ./bin/setup.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the hoodie@24.3.0 postinstall script 'node ./bin/setup.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the hoodie package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./bin/setup.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs hoodie
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls hoodie
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/bruno/workspace/test_hoodie_cli_2/npm-debug.log
```

If I try

```
 npm install --save hoodie@24.2.5
```

It works...

### PR DESCRIPTION
**Pull request by brunopedroso**
fixes #615

### CODE REVIEW

**Comment by gr2m** [comment]
you have to add the `fixes #615` to the PR description, the title alone won’t do (unless they changed that recently?). I added it for ya

File: bin/setup.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ if (saveRequested) {
       // - https://nodejs.org/api/path.html#path_path_parse_path
       var base = path.parse(pathToAppRoot).base
       var readMeContents = '# ' + base + '\n' + 'Created with [hoodie](https://github.com/hoodiehq)'
-      fs.writeFile(fd, readMeContents,
+      fs.writeFile(path.join(pathToAppRoot, 'README.md'), readMeContents,
```
Comment (gr2m) [inline bin/setup.js]: can we store `path.join(pathToAppRoot, 'README.md')` in a variable like `readmePath` and us it in both line 34 and 50?

**Review by gr2m — CHANGES_REQUESTED** [review]

File: bin/setup.js (hunk 1 above)
Comment (brunopedroso) [reply bin/setup.js]: there it is ;-)

**Review by gr2m — APPROVED** [review]

**Comment by gr2m** [comment]
thanks Bruno

**Comment by brunopedroso** [comment]
thank **you**

**Pull request merged**